develop/tutorials/articles/111-customizing/04-overriding-liferay-services.markdown

```diff
@@ -80,6 +80,13 @@ Each of these methods overrides a Liferay service method. These implementations
 merely add a few print statements that are executed before the original service
 implementations are invoked.
 
+Lastly, you need to add the following method to the bottom of your service wrapper so that it's able to find the appropriate service it's overriding on deployment.
+
+	@Reference(unbind = "-")
```

Comment:
hey @codyhoag  let me know if we need to update the project-templates-service-wrapper project template once this PR is reviewed.

Reply:
@gamerson I just got word back that we can move forward with this update. I'll be sending this for publication this week! I think the project template should also have this, as I think many developers would expect to have it by default.

Reply:
Thanks guys. Cody, I would agree with those sentiments.